app/controllers/application_controller.rb

```diff
@@ -6,9 +6,12 @@ class ApplicationController < ActionController::Base
                 :set_hotjar_site_id,
                 :prepare_cms
 
+  before_action :service_unavailable_page, if: :maintenance?
+
   helper_method :current_user,
                 :timeout_timer,
-                :debug?
+                :debug?,
+                :maintenance?
```

Comment:
We aren't using it in the view layer yet

Reply:
[37aa9b6](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/941/commits/37aa9b6ee52227bd8b6d9dd34c555eab44b37802)